qoqo/src/operations/mod.rs

```diff
@@ -147,6 +149,8 @@ pub fn operations(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<PhaseDisplacementWrapper>()?;
     m.add_class::<EchoCrossResonanceWrapper>()?;
     m.add_class::<PragmaAnnotatedOpWrapper>()?;
+    m.add_class::<ApplyConstantSpinHamiltonianWrapper>()?;
+    m.add_class::<ApplyTimeDependentSpinHamiltonianWrapper>()?;
```

Comment:
These two lines should also be moved a couple of lines down, in the section for the appropriate verrsion (1.9 or 1.10 depending on which of the PRs that are currently open is merged in first)